admin/class-yoast-notification-center.php

```diff
@@ -600,6 +600,12 @@ private function split_on_user_id( $notifications ) {
 	 * @return void
 	 */
 	public function update_storage() {
+		/**
+		 * Plugins might exit on the plugins_loaded hook.
+		 * This prevents the pluggable.php file from loading, as it's loaded after the plugins_loaded hook.
+		 * As we need functions defined in pluggable.php, make sure it's loaded.
+   		 */
+		require ABSPATH . WPINC . '/pluggable.php';
```

Comment:
```suggestion
		require_once ABSPATH . WPINC . '/pluggable.php';
```

Reply:
wow, i can't imagine i've missed that... im sorry, suggestion accepted!